loadtest/mock-oauth-cfn.yml

```diff
@@ -119,28 +119,23 @@ Resources:
           }
 
           const HANDLERS = {
-
-            'GET:/config': function(event, context, callback) {
-              return callback(null, {
-                statusCode: 200,
-                headers: {
-                  "content-type": "application/json"
-                },
-                body: '{ "browserid": { "issuer": "${MockIssuer}" } }'
-              });
-            },
-
             'POST:/v1/verify': function(event, context, callback) {
               try {
                 const token = JSON.parse(event.body).token;
                 const mockResponse = JSON.parse(token);
+                const mockStatus = mockResponse.status || 200;
+                const mockBody = mockResponse.body || {};
+                // Ensure successful responses always come from the mock issuer.
```

Comment:
Hmm, I don't understand how this comment relates to the block below...are we only including `issuer` in the body because, for failed responses, their issuer doesn't matter?

Reply:
I've reworded to try to make it clearer.